spec/component/models/activity_insight_oa_file_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'component/component_spec_helper'
+
+RSpec.describe ActivityInsightOaFile, type: :model do
+  subject { described_class.new }
+
+  it { is_expected.to have_db_column(:location).of_type(:string) }
+  it { is_expected.to have_db_column(:checksum).of_type(:string) }
+  it { is_expected.to have_db_column(:created_at).of_type(:datetime).with_options(null: false) }
+  it { is_expected.to have_db_column(:updated_at).of_type(:datetime).with_options(null: false) }
+
+  it { is_expected.to have_db_foreign_key(:publication_id) }
+  it { is_expected.to belong_to(:publication).inverse_of(:activity_insight_oa_files) }
+end
```

Comment:
Since we have a cascading delete on our relationship with publication, I'd like to specify that here. Normally I'd test this by creating an `ActivityInsightOaFile` record and associated `Publication`, deleting the `Publication`, and asserting that the `ActivityInsightOaFile` has been deleted also.